web/src/components/TtsHelpModal.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Link } from 'react-router-dom'
+import styled, { useTheme } from 'styled-components'
+
+import { BookIcon, WarningIcon } from '../assets'
+import Modal from './Modal'
+import Icon from './base/Icon'
+
+const StyledLink = styled(Link)`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 8px;
+  text-decoration: none;
+  color: black;
+`
+
+const ModalContent = styled.div`
+  padding: 0 16px 16px 48px;
+`
+
+const StyledWarningText = styled.div`
+  font-family: ${props => props.theme.fonts.web.contentFont};
+  font-size: 12px;
+  width: 72%;
```

Comment:
can you explain why you use this width? Looks quite random

Reply:
because it's a multiple of 4 number 😅

Reply:
what is more annoying is the font-size is smaller now because it was 14 /4 = 3.5 so I had to move it to 12.
I'm thinking of changing it to 16px..

Reply:
The multiples of four is mostly for the size of elements and empty spaces between them, a font-size of 14px is totally fine.